src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -160,7 +159,6 @@ private function decideConsensus(TokenInterface $token, array $attributes, $obje
                     break;
 
                 default:
```

Comment:
In the case of switch, it's better to keep the default, even if it has no operations (PSRs seems to not define this) - it implicitly defines that we do nothig (and din't forget adding it).